TelligentLocalInstance/TelligentLocalInstance.psd1

```diff
@@ -12,7 +12,7 @@
 ModuleToProcess = ''
 
 # Version number of this module.
-ModuleVersion = '0.0.4'
+ModuleVersion = '0.0.2'
```

Comment:
It will need to be increased when we come to publish anyway so does it matter at this point?